tests/python/frontend/onnx/test_forward.py

```diff
@@ -4418,317 +4419,323 @@ def verify_eyelike(indata):
+    "test_softmax_axis_0_expanded",
+    "test_softmax_axis_1",
+    "test_softmax_axis_1_expanded",
+    "test_softmax_axis_2_expanded",
+    "test_softmax_default_axis",
+    "test_softmax_default_axis_expanded",
+    "test_softmax_example_expanded",
+    "test_softmax_large_number_expanded",
+    "test_softmax_negative_axis_expanded",
+    "test_split_variable_parts_1d",
+    "test_split_variable_parts_2d",
+    "test_split_variable_parts_default_axis",
+    "test_split_zero_size_splits",
+    "test_squeeze",
+    "test_squeeze_negative_axes",
+    "test_strnormalizer_export_monday_casesensintive_lower",
+    "test_strnormalizer_export_monday_casesensintive_nochangecase",
+    "test_strnormalizer_export_monday_casesensintive_upper",
+    "test_strnormalizer_export_monday_empty_output",
+    "test_strnormalizer_export_monday_insensintive_upper_twodim",
+    "test_strnormalizer_nostopwords_nochangecase",
+    "test_tfidfvectorizer_tf_batch_onlybigrams_skip0",
+    "test_tfidfvectorizer_tf_batch_onlybigrams_skip5",
+    "test_tfidfvectorizer_tf_batch_uniandbigrams_skip5",
+    "test_tfidfvectorizer_tf_only_bigrams_skip0",
+    "test_tfidfvectorizer_tf_onlybigrams_levelempty",
+    "test_tfidfvectorizer_tf_onlybigrams_skip5",
+    "test_tfidfvectorizer_tf_uniandbigrams_skip5",
+    "test_training_dropout",
+    "test_training_dropout_default",
+    "test_training_dropout_default_mask",
+    "test_training_dropout_mask",
+    "test_training_dropout_zero_ratio",
+    "test_training_dropout_zero_ratio_mask",
+    "test_unique_sorted_with_axis",
+    "test_unique_sorted_with_axis_3d",
+    "test_unique_sorted_with_negative_axis",
+    "test_unsqueeze_axis_0",
+    "test_unsqueeze_axis_1",
+    "test_unsqueeze_axis_2",
+    "test_unsqueeze_negative_axes",
+    "test_unsqueeze_three_axes",
+    "test_unsqueeze_two_axes",
+    "test_unsqueeze_unsorted_axes",
+    "test_upsample_nearest",
 ]
 
 
-targets = [tgt for (tgt, _) in tvm.testing.enabled_targets()]
-
 target_skips = {
     "cuda": [
-        "test_mod_mixed_sign_float16/",
-        "test_qlinearconv/",
-        "test_resize_upsample_sizes_nearest/",
+        "test_basic_convinteger",
+        "test_convinteger_with_padding",
+        "test_range_float_type_positive_delta_expanded",
+        "test_range_int32_type_positive_delta_expanded",
+        "test_mod_mixed_sign_float16",
+        "test_qlinearconv",
+        "test_resize_upsample_sizes_nearest",
     ]
 }
 
 
-@pytest.mark.parametrize("target", targets)
-@pytest.mark.parametrize("test", onnx_test_folders)
-def test_onnx_nodes(test, target):
-    if target in target_skips:
-        for failure in target_skips[target]:
-            if failure in test:
-                pytest.skip()
-                break
-    for failure in unsupported_onnx_tests:
-        if failure in test:
-            pytest.skip()
-            break
+@pytest.mark.parametrize("onnx_test", onnx_test_folders)
+def test_onnx_nodes(target, dev, onnx_test):
```

Comment:
I'm not seeing how target and dev get into the test?

Reply:
This was a feature that I added in #8010 , where any unit test or fixture that accepts a `target` or `dev` argument is parametrized over all targets in `tvm.testing.enabled_targets`.  Basically, the same as the explicit `pytest.mark.parametrize` that was there previously, or the `tvm.testing.parametrize_targets`, but without needing each function to reproduce the same boilerplate.  The `target` parametrize is defined in [`tvm.testing._auto_parametrize_target`](https://github.com/apache/tvm/blob/main/python/tvm/testing.py#L788), and the `dev` fixture based on it is in the [`conftest.py`](https://github.com/apache/tvm/blob/main/conftest.py#L29).